packages/syntax-core/src/Tabs/TabLink.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  forwardRef,
+  type HtmlHTMLAttributes,
+  type ComponentProps,
+} from "react";
+import TabInternal from "./TabInternal";
+import classnames from "classnames";
+import styles from "./TabLink.module.css";
+
+type TabLinkProps = ComponentProps<typeof TabInternal> & {
+  /**
+   * Test id for the button
+   */
+  "data-testid"?: string;
+  /**
+   * The link that the Tab should route to.
+   *
+   */
+  href: string;
+  /**
+   * The target attribute specifies where to open the linked document.
+   *
+   */
+  target?: "_blank" | "_self" | "_parent" | "_top";
+  /**
+   * The rel attribute specifies the relationship between the document and the link.
+   *
+   */
+  rel?: HtmlHTMLAttributes<HTMLAnchorElement>["rel"];
+  /**
+   * An optional onClick event. This is used for certain wrapper's support (such as react-router-dom).
+   */
+  onClick?: React.MouseEventHandler<HTMLAnchorElement>;
+};
+
+const TabLink = forwardRef<HTMLAnchorElement, TabLinkProps>(
```

Comment:
a reminder for when you do the syntax bump for this pr to also create expressTabButton + NextTabButton + update the eslint rules to do no restricted imports for this component